src/thunks/cleanTrashedGroups.js

```diff
@@ -33,6 +34,7 @@ const removeGroupFromAllContacts = async (client, groupId) => {
           }
         }
       })
+      .indexFields(['relationships.groups.data'])
   )
 
```

Comment:
This is unrelated to the PR, but those queries should probably be moved in the dedicated file `src/helper/queries.js`